okio/src/commonTest/kotlin/okio/CommonOptionsTest.kt

```diff
@@ -16,20 +16,19 @@
 package okio
 
 import okio.ByteString.Companion.encodeUtf8
-import okio.TestUtil.bufferWithSegments
-import org.assertj.core.api.Assertions.assertThat
-import org.junit.Test
+import kotlin.test.Test
+import kotlin.test.assertEquals
 import kotlin.test.fail
 
-class OptionsTest {
+class CommonOptionsTest {
```

Comment:
Moving this test class to Kotlin/Common was pretty straight forward. I can create a copy instead and leave the original if we want.